console/client/src/theme/create-components.js

```diff
@@ -0,0 +1,295 @@
+import {
+  createTheme,
+  filledInputClasses,
+  inputLabelClasses,
+  outlinedInputClasses,
+  paperClasses,
+  tableCellClasses,
+} from '@mui/material'
+
+// Used only to create transitions
+const muiTheme = createTheme()
+
+export const createComponents = (config) => {
+  const { palette } = config
+
+  return {
+    MuiAvatar: {
```

Comment:
Out of curiosity, what is this giant data object?

Reply:
I think it's setting standard theme properties on the components. Like default border radii on a button, etc.